https://github.com/tigertv/secretpy/pull/7/checks?check_run_id=2685690496
```python
tests/test_scytale.py:5: in <module>
    from secretpy import Scytale
E   ImportError: cannot import name 'Scytale' from 'secretpy' (/home/runner/work/secretpy/secretpy/secretpy/__init__.py)
```
https://github.com/tigertv/secretpy/pull/7/checks?check_run_id=2685690386
```python
./secretpy/ciphers/scytale.py:10:25: E999 SyntaxError: invalid syntax
    def __enc(self, text: str, key: int):
                        ^
1     E999 SyntaxError: invalid syntax
```

--- a/secretpy/__init__.py
+++ b/secretpy/__init__.py
@@ -9,7 +9,8 @@ from .ciphers import (
     ColumnarTransposition, FourSquare, Gronsfeld, Keyword,
     MyszkowskiTransposition, Nihilist, SimpleSubstitution,
     Playfair, Porta, Polybius, Rot13, Rot18, Rot47, Rot5,
-    ThreeSquare, Trifid, TwoSquare, Vigenere, Vic, Zigzag)
+    Scytale, ThreeSquare, Trifid, TwoSquare, Vigenere, Vic,
+    Zigzag)
 # import alphabets
 # from .cmdecorators import *
 
@@ -21,6 +22,6 @@ __all__ = [
     "Chao", "ColumnarTransposition", "FourSquare", "Gronsfeld",
     "Keyword", "MyszkowskiTransposition",
     "Nihilist", "SimpleSubstitution", "Playfair", "Porta", "Polybius",
-    "Rot13", "Rot18", "Rot47", "Rot5", "ThreeSquare",
+    "Rot13", "Rot18", "Rot47", "Rot5", "Scytale", "ThreeSquare",
     "Trifid", "TwoSquare", "Vigenere", "Vic", "Zigzag",
 ]

--- a/secretpy/ciphers/__init__.py
+++ b/secretpy/ciphers/__init__.py
@@ -25,6 +25,7 @@ from .rot13 import Rot13
 from .rot18 import Rot18
 from .rot47 import Rot47
 from .rot5 import Rot5
+from .scytale import Scytale
 from .three_square import ThreeSquare
 from .trifid import Trifid
 from .two_square import TwoSquare
@@ -38,6 +39,6 @@ __all__ = [
     "Chao", "ColumnarTransposition", "FourSquare", "Gronsfeld",
     "Keyword", "MyszkowskiTransposition",
     "Nihilist", "SimpleSubstitution", "Playfair", "Porta", "Polybius",
-    "Rot13", "Rot18", "Rot47", "Rot5", "ThreeSquare",
+    "Rot13", "Rot18", "Rot47", "Rot5", "Scytale", "ThreeSquare",
     "Trifid", "TwoSquare", "Vigenere", "Vic", "Zigzag",
 ]

--- a/secretpy/ciphers/scytale.py
+++ b/secretpy/ciphers/scytale.py
@@ -1,19 +1,22 @@
 #!/usr/bin/python
 # -*- encoding: utf-8 -*-
 
+import secretpy.alphabets as al
+
 
 class Scytale:
     """
     The Scytale Cipher
     """
+    __alphabet = al.ENGLISH
 
-    def __enc(self, text: str, key: int):
+    def __enc(self, text, key, alphabet):
         block_list = []
         ans = ""
         rows = 0
 
         # Round up number of "rows"
-        # "ceil(len(text) / key" not possible due to compatibility to python 2)
+        # "ceil(len(text) / key" not possible due to compatibility to python 2
         if (len(text) % key) != 0:
             rows = int((len(text) / key) + 1)
         else:
@@ -35,7 +38,7 @@ class Scytale:
                     break
         return ans
 
-    def __dec(self, text: str, key: int):
+    def __dec(self, text, key, alphabet):
         block_list = []
         ans = ""
         rows = 0
@@ -44,7 +47,7 @@ class Scytale:
         block_end = 0
 
         # Round up number of "rows" and "block_end"
-        # "ceil(len(text) / key" not possible due to compatibility to python 2)
+        # "ceil(len(text) / key" not possible due to compatibility to python 2
         if (len(text) % key) != 0:
             rows = int((len(text) / key) + 1)
             block_end = int((len(text) / key) + 1)
@@ -84,26 +87,30 @@ class Scytale:
                     break
         return ans
 
-    def encrypt(self, text, key):
+    def encrypt(self, text, key, alphabet=None):
         """
         Encryption method
         :param text: Text to encrypt
         :param key: Encryption key - Number of windings
+        :param alphabet: Alphabet which will be used,
+                         if there is no a value, English is used
         :type text: string
         :type key: integer
         :return: encrypted text
         :rtype: string
         """
-        return self.__enc(text, key)
+        return self.__enc(text, key, alphabet)
 
-    def decrypt(self, text, key):
+    def decrypt(self, text, key, alphabet=None):
         """
         Decryption method
         :param text: Text to decrypt
         :param key: Decryption key - Number of windings
+        :param alphabet: Alphabet which will be used,
+                         if there is no a value, English is used
         :type text: string
         :type key: integer
         :return: decrypted text
         :rtype: string
         """
-        return self.__dec(text, key)
+        return self.__dec(text, key, alphabet)

